### PR TITLE
Fixing typos (an List -> a List)

### PR DESCRIPTION
**Pull request by danhyun**
Hello from GR8Conf EU 2015

### CODE REVIEW

File: subprojects/groovy-jmx/src/main/groovy/groovy/jmx/builder/JmxListenerFactory.groovy [hunk 1]
```diff
@@ -1,81 +1,81 @@
...
+
+    public boolean isLeaf() {
+        return true;
+    }
+
+    public void onNodeCompleted(FactoryBuilderSupport builder, Object parentNode, Object thisNode) {
+        if (parentNode != null) {
+            parentNode.add(thisNode)
+        }
+    }
```
Comment (PascalSchumacher) [inline subprojects/groovy-jmx/src/main/groovy/groovy/jmx/builder/JmxListenerFactory.groovy]: Thanks for the contribution. :+1: 

Could you please revert the change (format/line endings?) above, so it's obvious that you corrected just the "an" typos?

Thanks again.

**Comment by PascalSchumacher** [comment]
Greeting to GR8Conf EU 2015. :)

File: subprojects/groovy-jmx/src/main/groovy/groovy/jmx/builder/JmxListenerFactory.groovy (hunk 1 above)
Comment (danhyun) [inline subprojects/groovy-jmx/src/main/groovy/groovy/jmx/builder/JmxListenerFactory.groovy]: Yes, I was running this from a talk and noted the line endings and thought I would be scolded :P

It also served as a lesson to be polite to project maintainers

File: subprojects/groovy-jmx/src/main/groovy/groovy/jmx/builder/JmxListenerFactory.groovy [hunk 2]
```diff
@@ -22,7 +22,7 @@ import javax.management.InstanceNotFoundException
 import javax.management.NotificationFilterSupport
 
 /**
- * This factory class is used to create an listener() node for JmxBuilder.  Listener nodes are used
+ * This factory class is used to create a listener() node for JmxBuilder.  Listener nodes are used
```
Comment (danhyun) [inline subprojects/groovy-jmx/src/main/groovy/groovy/jmx/builder/JmxListenerFactory.groovy]: As intended :)

**Comment by danhyun** [comment]
Line endings should be unmodified now

**Comment by PascalSchumacher** [comment]
Thanks! :+1:

**Comment by danhyun** [comment]
Thank you! Thanks for helping me demonstrate how to contribute to Groovy :D

**Comment by bodiam** [comment]
Great first commit! Now keep them coming!
